src/js/server/utils.js

```diff
@@ -0,0 +1,188 @@
+/*
+
+    Utilities methods for performing user operations. These utilities should
+    be usable from both http endpoints, and any other locations that can
+    consume promise based API's.
+
+    Utilities are included for:
+    - Creating new user accounts
+    - Unlocking user accounts with a username/email and password.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var request = require("request"); // TODO:  Replace this with a writable data source.
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.express.user.utils");
+
+/**
+ *  gpii.express.user.utils
+ *
+ *  Component that implements backend utilities as promise based
+ *  invokers that can be used from http handers or other infrastructure.
+ *
+ *  Expects to be distributed the same couch options as the other components
+ *  in gpii-express-user.
+ */
+fluid.defaults("gpii.express.user.utils", {
```

Comment:
Utils is far too generic and uninformative a name. Given this component appears to deal mainly with passwords and their encoding I suggest something like "crypto" or "passwords"